website/static/static/js/version.js

```diff
@@ -1,5 +1,5 @@
 const presto_latest_presto_version = '0.224';
-const presto_latest_num_commits = 114;
-const presto_latest_num_contributors = 33;
-const presto_latest_num_committers = 14;
+const presto_latest_num_commits = 79;
+const presto_latest_num_contributors = 21;
```

Comment:
How is this number calculated? How is it presented?
Is it calculated with `NUM_CONTRIBUTORS=`${GIT_LOG} --format='%aE' | sort | uniq | wc -l | awk '{$1=$1;print}'``?

`git log 0.223..0.224 --format='%aE' | sort -u` includes myself, even though I did not  contribute anything to 0.224 release.